test/AlixBa/Addic7edSubtitles/Helpers/EpisodeTest.php

```diff
@@ -10,9 +10,9 @@ function testCapitalizedFileName()
 
         $this->assertEquals('The.Big.Bang.Theory', $episode->showName);
         $this->assertEquals('thebigbangtheory', $episode->sanitizedShowName);
-        $this->assertEquals('08', $episode->season);
-        $this->assertEquals('18', $episode->ep);
-
+        $this->assertEquals(8, $episode->season);
+        $this->assertEquals(18, $episode->ep);
+        //$this->assertEquals(['lol'], $episode->groups); // XXX shouldn't this contain only lol ?
         $this->assertEquals(['hdtv', 'x264'], $episode->tags);
```

Comment:
the thing is addicted groups and file group does not always match.

For instance `The.Big.Bang.Theory.S08E18.HDTV.x264-LOL.mp4` will match LOL group, and on addicted nothing will.
![capture d ecran 2015-04-05 a 17 42 14](https://cloud.githubusercontent.com/assets/2699647/6997428/2917e96c-dbbb-11e4-9daf-b776943c8dbd.png)

But I'm pretty sure (90%) that dimension will be perfectly sync for this file. 
It happens to me a lot. That's why I prefer to use substitution release groups (based on format type).